securedrop/sass/modules/_flash.sass

```diff
@@ -16,27 +17,33 @@
       font-size: 18pt
 
     &.success
-      border: 1px solid #E1F1E5
-      background-color: #E6FFEB
+      background-color: $color_success_background
 
-      strong
-        color: #299549
+      b
+        color: $color_success_text_strong
+        padding-right: 0.5em
 
     &.notification
-      border: 1px solid #8ed9f6
-      background-color: #e3f7fc
+      background-color: $color_notification_background
```

Comment:
Because this is a shared module, these changes bleed into the Source Interface, while the icon changes do not. As a consequence, some Source Interface messages are now a hybrid between the old style and the new style. Case in point (rendered without border but with old icon):

![Screenshot from 2021-02-17 16-52-29](https://user-images.githubusercontent.com/213636/108287968-95ccba80-7140-11eb-8235-4ed06c4a7d30.png)

Since we are updating the shared style, I think it would be reasonable to update the impacted icon(s) in the Source Interface flash template as well, and make sure that all notifications look how we want them to.

Reply:
I'm actually gonna split them for now - all old style for source, new style for Journalist - as a) the SI has different notification states, so we may not want to use the same icons, b) there would probably be text updates needed in there too, and c) if there are text updates, we're adding even more to translator workflow for this release. Should also reduce release QA reqs.

If there isn't already a ticket for improving the SI flash messaging, we should put one in. Otherwise this is just scope creep, and a bit late in the process for it.